src/main/java/seedu/java/util/Parser.java

```diff
@@ -0,0 +1,74 @@
+package seedu.java.util;
+
+/**
+ * All static method. Handles input from user.
+ * readCommand(String)
+ * readTask(String)
+ * readTiming(String)
+ * readNum(String)
+ */
+public class Parser {
+    /**
+     * Static method. Returns a Command (inferred instruction)
+     * @param input from UI
+     * @return Command
+     * @throws Exception
+     */
+    public static Command readCommand(String input) throws Exception {
```

Comment:
I think it is a better idea to throw a specific Exception, rather that throwing a general `Exception` here

Throwing `Exception` would cause any Exceptions to be typecast to that, so any method that uses `readCommand` will be forced to catch `Exception` generally (and it would be difficult to determine the true source of the Exception) - this would necessitate the "Pokemon" Exception catching that reviewer _yjskrs_ mentioned earlier!